testsuite/tests/apicast/policy/batcher/test_batcher_policy_mapping_rules.py

```diff
@@ -67,4 +69,4 @@ def test_batcher_policy_append(client, application):
     sleep(50)
 
     usage_after = analytics.list_by_service(application["service_id"], metric_name="hits")["total"]
-    assert usage_after == usage_before + 4
+    assert usage_after == usage_before + 3
```

Comment:
Why was the usage_after changed?

Reply:
Because before the change there were 3 hits from test and one from test setup which is now correctly handled, hence only 3 hits from test left.